src/main/scala/io/dronekit/cloud/AWSWrapper.scala

```diff
@@ -170,7 +170,7 @@ class AWSWrapper(S3Client: AmazonS3Client = S3.client)
   }
 
   def streamUpload(s3url: S3URL): Sink[ByteString, Future[CompleteMultipartUploadResult]] = {
-    multipartUploadTransform(s3url).to(Sink.head)
+    multipartUploadTransform(s3url).fold(Seq.empty[UploadPartResult])(_ :+ _).to(Sink.head)
```

Comment:
If you wanted to keep the `UploadPartResult`s, you would use `multipartUploadTransform`. This method doesn't currently expose the stream output because of the `Sink.head`, right?